app/views/groups/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="container mt-4">
+<h3 class="category">CATEGORY</h3>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give exact design as [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) to your `Home` page's all elements so you should give correct style to `Navbar` becasue there disappear `Log Out` button and elements locate opposite side and created `categories` texts appear big and also compositions are bit wrong placed, lastly give correct design to `Add New Category` button 🙏🏼    

![image](https://github.com/YoutLone/Budget_App/assets/63915024/9b162f15-ff78-4525-b70d-b7f989e2744b)